metricq/types.py

```diff
@@ -50,9 +50,7 @@ def from_string(duration_str: str):
         m = re.fullmatch(r"\s*([+-]?\d*[.,]?\d+)\s*([^\d]*)\s*", duration_str)
         if not m:
             raise ValueError(
-                'invalid duration string {}, not of form "number unit"'.format(
-                    duration_str
-                )
+                'invalid duration string {}, not of form "number unit"'.format(duration_str)
```

Comment:
shouldn't be allowed as per the project black configuration (in `pyproject.toml`)

Reply:
corrected